pyscf/pbc/cc/kccsd_rhf.py

```diff
@@ -628,7 +626,9 @@ def ccsd_t(self, t1=None, t2=None, eris=None):
         from pyscf.pbc.cc import kccsd_t_rhf
         if t1 is None: t1 = self.t1
         if t2 is None: t2 = self.t2
-        if eris is None: eris = self.ao2mo(self.mo_coeff)
+        if eris is None:
+            eris = self.ao2mo(self.mo_coeff)
+            self.e_hf = self.get_e_hf()
```

Comment:
self.e_hf may not be created if eris is not None

Reply:
I've deleted the self.e_hf evaluation here.